.github/workflows/pull-request.yml

```
@@ -42,7 +42,11 @@ jobs:
     needs: ['install', 'build']
 
     steps:
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)